docusaurus.config.ts

```diff
@@ -206,10 +206,10 @@ const config = (async (): Promise<Config> => {
           }),
         ],
       },
-      announcementBar: {
-        id: 'innovation-day',
-        content: `📢 Join us virtually for <b>wasmCloud Innovation Day</b> on September 18! <a href="/innovation-day">Register now</a> for a day of immersive learning, demos, user talks, and more.`,
-      },
+//      announcementBar: {
+//        id: 'innovation-day',
+//        content: `📢 Join us virtually for <b>wasmCloud Innovation Day</b> on September 18! <a href="/innovation-day">Register now</a> for a day of immersive learning, demos, user talks, and more.`,
+//      },
```

Comment:
```suggestion
```

We could probably remove this right?

Reply:
Think I prefer to leave it in so anyone can quickly uncomment and add an announcement bar back without looking up the syntax, but maybe we should make the contents more generic/instructive? I'll update in that direction